src/assets/i18n/en_US.json

```diff
@@ -275,17 +275,19 @@
   "account.overview.wishlist.view_all": "View All Wish Lists",
   "account.password.error.regexp": "Your password must be at least 7 characters in length.",
   "account.password.label": "Current Password",
-  "account.payment.further_payments.heading": "Further Saved Payment Instruments",
+  "account.payment.further_payments.heading": "Further Payment Methods",
   "account.payment.heading": "Saved Payment Information",
   "account.payment.link": "Payment",
-  "account.payment.message": "Select a preferred payment instrument for quicker checkout on future purchases.",
-  "account.payment.no_entries": "Currently, you do not have any saved payment instruments.",
-  "account.payment.payment_created.message": "Your preferred payment instrument has been saved to your account.",
+  "account.payment.message": "Select a preferred payment method for quicker checkout on future purchases.",
+  "account.payment.no_entries": "Currently, you do not have any payment methods.",
```

Comment:
No comma. See comment for DE: Is "saved" missing?
Else maybe write "Currently there are no payment methods."